lib/controllers/register.js

```diff
@@ -89,6 +89,20 @@ function applyDefaultAccountFields(config,req) {
     req.body.surname = 'Anonymous';
   }
 }
+/**
```

Comment:
The `loginResponder` is in `helpers/` so I think that's a good place for this :)